portfolio/src/main/webapp/script.js

```diff
@@ -66,13 +61,15 @@ function getComment() {
 function createCommentElement(text) {
   const commentElement = document.createElement('li');
   commentElement.className = 'comment';
-
   const textElement = document.createElement('span');
   textElement.innerText = text;
-  
   commentElement.appendChild(textElement);
   return commentElement;
 }
 
-
+/** Clears out old comments and inserts the selected number of comments. */
+function changeCommentsNum(){
```

Comment:
What is this function? Where is it used? 

Also, add space between punctuation marks: `changeCommentsNum() {...}`